components/icon-buttons.jsx

```diff
@@ -1,16 +1,32 @@
 var React = require('react');
+var classnames = require('classnames');
 
 var IconButtons = React.createClass({
+  propTypes: {
+    children: React.PropTypes.object.isRequired
+  },
   render: function() {
+    let children = this.props.children.filter(child => !!child);
+    let numChildren = children.length;
+    // Bootstrap uses 12-col layout.
+    // We want icon buttons to fill up the entire row when possible
+    // Let's calculate how many columns each button should occupy
+    let numColSpan = Math.floor(12 / numChildren);
+    // if buttons can't fill up the entire row, add offset to the first button
+    // so the group of buttons looks centered in the row
+    let colOffset = Math.floor((12 - numColSpan*numChildren) / 2);
```

Comment:
Not sure if we need this as we can't guarantee group of buttons *can really be* positioned at center. e.g., when `numChildren` is 9 `colOffset` will be `1` instead of `1.5`(which is a number that doesn't make sense in Bootstrap's world)